osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -496,6 +496,7 @@ public override void OnEntering(IScreen last)
             FilterControl.Activate();
 
             ModSelect.SelectedMods.BindTo(selectedMods);
+            Scheduler.AddDelayed(Carousel.StartAllowingSelectionOnFiltered, 250);
```

Comment:
this whole thing is awkward

Reply:
This is meant to wait for all the automatic beatmap bindings to not change values anymore before allowing beatmap carousel to react to user changes to the filter. Every call to filter function before actually entering song select is automatic and not issued by the user.

idk if there is a better way to do this.